config/schedule.rb

```diff
@@ -26,63 +26,31 @@
 
 every 1.day, at: '1:15am', roles: [:cron_prod1] do
   browse_facet_update(
-    'browse:call_numbers',
-    solr_url: 'http://lib-solr3.princeton.edu:8983/solr/catalog-production',
-    output: '/tmp/cron_log.log'
-  )
-end
-
-every 1.day, at: '6:20am', roles: [:cron_prod1] do
-  browse_facet_update(
-    'browse:load_call_numbers',
+    'browse:call_numbers:update',
```

Comment:
These need to be updated for the incremental update requests.  Should the timestamp be used to query for documents updated 24 hour prior?